h2o-py/tests/testdir_algos/automl/pyunit_automl_dummy_steps.py

```diff
@@ -0,0 +1,64 @@
+from __future__ import print_function
+import os
+import sys
+import time
+
+sys.path.insert(1, os.path.join("..","..",".."))
+import h2o
+from tests import pyunit_utils as pu
+from h2o.automl import H2OAutoML
+
+"""This test suite checks the AutoML dummy modeling steps"""
+
+
```

Comment:
Idea is not to test the DummyStep, but use it for our tests relying on timeout logic.

Reply:
@sebhrusen I think that having tests for our custom tool, that tests with timeout logic are going to rely on, is a necessary and a good thing. Within a scope of this PR I would expect exactly this kind of tests.

Reply:
having test is always fine, just saying that as the Jira ticket mentions a lot the question of testing, the goal is to use it for specific tests

Reply:
I agree that these tests are probably not necessary, I wrote them mainly for me as a sanity check. If necessary, I will remove them before merging but until this PR reaches the state when I will know I don't have to debug/modify DummySteps logic I would keep it here just to make it easier for me...